packages/eslint-plugin-lit-a11y/lib/rules/aria-attr-types.js

```diff
@@ -59,8 +77,7 @@ const validityCheck = (value, expectedType, permittedValues) => {
     case 'integer':
     case 'number':
       // Booleans resolve to 0/1 values so hard check that it's not first.
-      // eslint-disable-next-line no-restricted-globals
-      return typeof value !== 'boolean' && isNaN(Number(value)) === false;
+      return typeof value !== 'boolean' && Number.isNaN(Number(value)) === false;
```

Comment:
Was previously globalThis.isNaN, but if we're explicitly casting with `Number()`, then `Number.isNaN()` should be fine, right?

Reply:
should be fine I think 👍